src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/samples/Sample7_ConfiguringPipeline.cs

```diff
@@ -20,29 +20,26 @@ public partial class ConfigurationSamples
         public async Task ConfiguringPipeline()
         {
             // this instance will hold pipeline creation options
-            var options = ConfigurationClient.CreateDefaultPipelineOptions();
+            var options = new HttpPipelineOptions();
 
             // specify custon HttpClient
-            options.Transport = new HttpClientTransport(s_client);
-
-            // remove logging policy
-            options.LoggingPolicy = null; 
```

Comment:
We need a way to remove the default policies that will be added inside client. How do you envision this working?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/5524#discussion_r267514079